core/storage/chain/impl/chain_store_impl.hpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_STORAGE_CHAIN_CHAIN_STORE_IMPL_HPP
+#define CPP_FILECOIN_CORE_STORAGE_CHAIN_CHAIN_STORE_IMPL_HPP
+
+#include <map>
+
+#include "blockchain/block_validator.hpp"
+#include "blockchain/weight_calculator.hpp"
+#include "common/logger.hpp"
+#include "common/outcome.hpp"
+#include "crypto/randomness/chain_randomness_provider.hpp"
+#include "crypto/randomness/randomness_types.hpp"
+#include "primitives/cid/cid.hpp"
+#include "storage/chain/chain_data_store.hpp"
+#include "storage/chain/chain_store.hpp"
+#include "storage/ipfs/impl/ipfs_block_service.hpp"
+
+namespace fc::storage::blockchain {
+
+  /**
+   * @struct MmCids is a struct for using in mmCache cache
```

Comment:
What is mm and mmcache?

Reply:
It is required for caching implementation,  to be used in FIL-155, I took names from lotus.

Reply:
It is better to have meaningful and descriptive names, so if you know what mmcache means, please provide description. I . managed to google only PHP-related stuff, but I hope it is not our case.